Sources/App/WebView/WebViewController.swift

```diff
@@ -24,6 +24,7 @@ class WebViewController: UIViewController, WKNavigationDelegate, WKUIDelegate, U
 
     var keepAliveTimer: Timer?
     private var initialURL: URL?
+    private let authenticationService: AuthenticationServiceProtocol
```

Comment:
DI using `Current` (see https://www.pointfree.co/blog/posts/21-how-to-control-the-world)

Reply:
Good one! I was looking for which DI the project used, thanks!

Reply:
And of course that was my first reaction "Isn’t Current a singleton? Aren’t singletons evil?" haha like they have in the FAQ